pkg/kubelet/cm/dra/manager_test.go

```diff
@@ -54,6 +54,10 @@ func (s *fakeDRADriverGRPCServer) NodePrepareResources(ctx context.Context, req
 	if s.timeout != nil {
 		time.Sleep(*s.timeout)
 	}
+	// handle detectVersion
+	if len(req.Claims) == 0 {
+		return &drapbv1.NodePrepareResourcesResponse{}, nil
+	}
```

Comment:
This is very suspicious. Does it mean that all DRA plugins should implement something like this? Can it be a reason why ci test fails?

Reply:
I think no. I made that change because my fake server was assuming receiving claims and an index out of range error was returned due to the detect version function and the call I make with an empty request to determine whether the server is supporting v1alpha3 API or not